src/seedu/addressbook/data/person/Printable.java

```diff
@@ -0,0 +1,5 @@
+package seedu.addressbook.data.person;
+
```

Comment:
Add comments for interface. Especially when it contains no code, nobody would understand what this interface is supposed to do without comments.